perf-test/src/test/java/co/cask/cdap/db/DatabaseTestBase.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright © 2019 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.db;
+
+import co.cask.cdap.app.etl.ETLTestBase;
+import co.cask.cdap.common.UnauthenticatedException;
+import co.cask.cdap.datapipeline.SmartWorkflow;
+import co.cask.cdap.etl.proto.v2.ETLBatchConfig;
+import co.cask.cdap.etl.proto.v2.ETLStage;
+import co.cask.cdap.proto.ProgramRunStatus;
+import co.cask.cdap.proto.artifact.AppRequest;
+import co.cask.cdap.proto.id.ApplicationId;
+import co.cask.cdap.test.ApplicationManager;
+import co.cask.cdap.test.WorkflowManager;
+import com.google.common.collect.ImmutableMap;
+
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.PrintStream;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+
+public abstract class DatabaseTestBase extends ETLTestBase {
+
+  protected static PrintStream out;
+
+  static {
+    try {
+      String property = System.getProperty("database.metrics.file");
+
+      out = property == null ? System.out : new PrintStream(new FileOutputStream(property, false));
+    } catch (FileNotFoundException e) {
+      e.printStackTrace();
+    }
+  }
+
+
+  public void writeMetrics(Map<String, String> tags) throws IOException, UnauthenticatedException {
+
+    StringBuilder metrics = new StringBuilder();
+    List<String> strings = getMetricsClient().searchMetrics(tags);
```

Comment:
please don't name variables like this: "strings"

Reply:
done